install.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# VARIABLES, CHANGE AS NEEDED
+GITHUB_REPO="https://github.com/bmFtZQ/edge-frfox.git"
+PROJECT_NAME=$(basename $GITHUB_REPO | cut -d '.' -f 1)
+
+if [[ $OSTYPE == "darwin"* ]]; then
+  FIREFOX_DIR="$HOME/Library/Application Support/Firefox"
+else
+  FIREFOX_DIR=$HOME/.mozilla/firefox
+fi;
+
+PROFILE_ROOTDIR=$FIREFOX_DIR/$(grep -E "Path=.*\.(dev-edition-default|default-.*)" "$FIREFOX_DIR/profiles.ini" | tail -1 | cut -c 6-);
+OPTIONALS=(
+  "widget.macos.native-context-menus" "false"
+  "browser.theme.dark-private-windows" "false"
+)
+
+# COLORS
+GREEN='\033[0;32m'
+YELLOW='\033[0;93m'
+NC='\033[0m'
+CYAN='\033[0;36m'
+RED='\033[0;31m'
+
+# UTILITY FUNCTIONS
+set_pref() {
+  echo "setting $1 to $2 in user.js";
+  echo "user_pref(\"$1\", $2);" >> $PROFILE_ROOTDIR/user.js;
+}
+
+delete_pref() {
+  echo "resetting $(echo $1 | cut -d '"' -f 2) to default";
+  sed -i "/$1/d" "$PROFILE_ROOTDIR/user.js"
```

Comment:
This causes an error on macOS:

```
sed: 1: command a expects \ followed by text
```

Adding ` ''` after `-i` seems to fix it, but breaks when run on Linux with the GNU version of `sed`.

Here is a Stack Overflow question that contains more information: https://unix.stackexchange.com/questions/92895/how-can-i-achieve-portability-with-sed-i-in-place-editing